capstone/src/components/User/Experience/Experience.jsx

```diff
@@ -42,10 +42,10 @@ export default function Experience({ setIsLoggedIn, isLoggedIn }) {
     } else if (username != null) {
       Promise.all([getExperienceInfo(username), getAllFeedbackInfo()]).then(
         ([expRes, feedbackRes]) => {
-          // If there are no records on the experience that the user owns
+          // If there are no values for a field
           // send to that page to promote submission.
           // Otherwise, just set values in state
-          if (Object.keys(expRes.data).length === 0) {
+          if (Object.keys(expRes.data).every((experienceField) => !!experienceField)) {
```

Comment:
Is `activeFeedback` one of the experience fields? I noticed you initialized this to an array above. If so, arrays are always [truthy](https://developer.mozilla.org/en-US/docs/Glossary/Truthy). I

Reply:
Wow! I definitely didn't notice that!!!
I just added that to an issue I plan to address soon
Thank you so much for that one